### PR TITLE
Update 1.21

### PR DESCRIPTION
**Pull request by TonimatasDEV**
I added a new category named "Limbo".

### CODE REVIEW

File: java/LIMBO.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Minecraft Java Folia Plugins software.
+This list contains Minecraft Java plugins Limbo server softwares.
```
Comment (LeStegii) [inline java/LIMBO.md:2]: I'd remove the "Minecraft Java plugins" part as plugins are not really a limbo feature.

File: OTHERS.md [hunk 2]
```diff
@@ -10,11 +10,18 @@ This list contains softwares that are related to minecraft servers but are no se
 - **Platforms:** Bukkit/Spigot/Paper, Bungeecord, Sponge, Velocity
 
 ### [✴️ Minestom](https://github.com/Minestom/Minestom)
-- **Version:** 1.18 - 1.20.4
+- **Version:** 1.18 - 1.21
 - **Author:** Minestom
 - **Fork:** ?
```
Comment (LeStegii) [inline OTHERS.md:15]: I think this should be `-` instead, as the fork chain is known but it's simply not a fork. (Same thing for some other entries in this category)

**Review by LeStegii — CHANGES_REQUESTED** [review]
I really like the new category and that you put some softwares into the other category.

**Comment by TonimatasDEV** [comment]
Done but AnchorMC (Vanilla java category) repository was removed. I think we need to do a "LOST.md" category for servers of #24.

**Pull request merged**